recipe/recipe.yaml

```diff
@@ -1,49 +1,54 @@
-{% set name = "dask-awkward" %}
-{% set version = "2025.2.0" %}
-{% set python_min = "3.8" %}
+schema_version: 1
+
+context:
+  name: dask-awkward
+  version: 2025.2.0
+  python_min: 3.8
 
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: ${{ name|lower }}
+  version: ${{ version }}
 
 source:
-  url: https://pypi.org/packages/source/d/dask_awkward/dask_awkward-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/d/dask_awkward/dask_awkward-${{ version }}.tar.gz
   sha256: 665df70be4f0394e8fa4ef50fcd80272f72cef6a4784e7f935e091d093d9aa2a
 
 build:
+  number: 1
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: ${{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
-    - python {{ python_min }}
+    - python ${{ python_min }}.*
     - pip
     - hatchling >=1.8.0
     - hatch-vcs
     - toml
   run:
-    - python >={{ python_min }}
+    - python >=${{ python_min }}
     - awkward >=2.5.1
     - dask >=2023.04.0
     - cachetools
     - typing_extensions >=4.8.0
 
-test:
-  imports:
-    - dask_awkward
+tests:
+  - python:
+      imports:
+        - dask_awkward
+      pip_check: false
```

Comment:
```suggestion
      pip_check: true
```

Reply:
Ah, sorry, missed this suggestion. I committed the change already 👍